forwards_paper.tex

```diff
@@ -335,16 +351,18 @@ \subsection*{Recording the pedigree in forwards time}
 For example, in Algorithm~\algref{W} we would generate mutations after
 we have recorded the edges joining the new node $n$ to its parents
 $P_a$ and $P_b$ in step \algref{W7}.
-For example, if we assume that infinite sites mutations
-occur at rate $\mu$ per generation then there are $\mbox{Poisson}\left((r - \ell)(\tau_p
+For example, if we assume that mutations occur according to the infinite sites model
+at rate $\mu$ per generation per unit length then there are 
+$\mbox{Poisson}\left(\mu (r - \ell)(\tau_p
 - \tau_c)\right)$ mutations on each edge $(\ell, r, p, c)$. Each of these mutations
-will occur on a distinct site $x$ drawn uniformly from $(\ell, r]$ and be over node $c$.
```

Comment:
I keep wanting to write "above" instead of "over" for mutations and their nodes.  Apologies.  Feel free to change this back.